include/aws/common/byte_buf.h

```diff
@@ -432,14 +432,14 @@ bool aws_byte_cursor_eq_c_str_ignore_case(const struct aws_byte_cursor *const cu
  * Case-insensitive hash function for array containing ASCII or UTF-8 text.
  */
 AWS_COMMON_API
-uint64_t aws_hash_array_ignore_case(const void *array, size_t len);
+uint64_t aws_hash_array_ignore_case(const void *const array, const size_t len);
```

Comment:
Why are we doing this? See this post about why const here is meaningless and confusing:
https://abseil.io/tips/109

Reply:
Gzz you are right! I'll remove then from now on.